doc/API.md

```diff
@@ -176,6 +177,19 @@ returns information about the consensus set, such as the current block height.
 }
 ```
 
+#### /consensus/validate/transactionset [POST]
+
+validates a set of transactions using the current utxo set.
```

Comment:
Determines whether a transaction set is valid using the current state of the network. A valid transaction could be included in the next block if mined.